src/text_drop.rs

```diff
@@ -0,0 +1,52 @@
+extern crate gdk;
+extern crate gtk;
+
+use std::fs::File;
+use std::io::prelude::*;
+use std::string::String;
+
+use gtk::prelude::*;
+
+fn main() {
+    if gtk::init().is_err() {
+        println!("Failed to initialize GTK.");
+        return;
+    }
+
+    // Create a scrollable text view
+    let text_view = gtk::TextView::new();
+    text_view.set_wrap_mode(gtk::WrapMode::Char);
+    let scrolled_text_view = gtk::ScrolledWindow::new(None, None);
+    scrolled_text_view.set_policy(gtk::PolicyType::Automatic, gtk::PolicyType::Automatic);
+    scrolled_text_view.add(&text_view);
+
+    // Embed the scrolling textview into our window
+    let window = gtk::Window::new(gtk::WindowType::Toplevel);
+    window.add(&scrolled_text_view);
+    window.show_all();
+
+    let accepted = vec![
+        gtk::TargetEntry::new("text/uri-list", gtk::TargetFlags::empty(), 0),
+//        gtk::TargetEntry::new("text/plain", gtk::TargetFlags::empty(), 1),
+//        gtk::TargetEntry::new("STRING", gtk::TargetFlags::empty(), 2),
+    ];
+    text_view.drag_dest_set(gtk::DEST_DEFAULT_ALL,
+                            &accepted,
+                            gdk::ACTION_COPY | gdk::ACTION_MOVE); // Maybe this should be ACTION_ASK?
+    text_view.connect_drag_data_received(|_, d, _, _, s, tt, t| {
+        println!("-{}", tt);
```

Comment:
We're generally not big fans of printing things in stdout when using a graphic application. Use a small window or any equivalent which allows to remain in the application to see the info please.